phx/performances/performances_scraper.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import re
+from datetime import date, datetime
+
+import requests
+from athletes.models import Athlete
+from bs4 import BeautifulSoup
+from events.models import Event
+from performances.models import Performance
+
+logger = logging.getLogger(__name__)
+
+
+class PerformancesScraper:
+
+    ATHLETE_PROFILE_URL = "https://www.thepowerof10.info/athletes/" \
+                          "profile.aspx?athleteid={athlete_id}&viewby=date"
+
+    def __init__(self):
+        self.events = {}
+        self.performances = []
+        self.inactive_athletes = set[str]()
+
+    def find_performances(self, athlete: Athlete, since: date):
+        """
+        Search Power of 10 for all performances by the given athlete since the
+        given date
+
+        Returns the total number of performances found
+        """
+
+        if not athlete.power_of_10_id:
+            logger.warning(f"No Power of 10 ID for {athlete}")
```

Comment:
(I should know this)

Will these logs be stored in text files on the server, or are we just keeping them in for debugging during development? Either way it's not a problem, and good to keep in for reference

Reply:
I _think_ and am hoping that when the cron is set-up the logs will be stored to a file if we take the same approach you took with the previous cronjob i.e.

```
python manage.py runjobs hourly --settings=phx.settings.production >> /home/phx/crons.log
```

Reply:
(correction, I think only errors will go to the file)